src/sandstorm/app-index/indexer.c++

```diff
@@ -389,7 +394,7 @@ void Indexer::updateIndexInternal(kj::StringPtr outputFilename, kj::StringPtr ou
     kj::FdOutputStream(file.getFd()).write(text.begin(), text.size());
     file.finalize(kj::str(outputDir, "/", appEntry.first, ".json"));
 
-    if (approvedApps) {
+    if (!experimental) {
```

Comment:
I guess the idea is that we unconditionally write the app SPK file symlink to `/var/apps`, but since we only need to do it once, we skip that work `if (experimental)`?